src/Orc.Automation/ScenarioManagement/ScenarioManager.cs

```diff
@@ -0,0 +1,86 @@
+﻿namespace Orc.Automation.ScenarioManagement;
+
+using System;
+
+public static class ScenarioManager
```

Comment:
Static makes it hard to unit test. Do we really need this to be static?

Reply:
this class is used in Fody interseptors methods... so i wanted it to be initialized at this point - and be as simple as possible... maximum that i can do in this situation is to use Singleton...and we don't have test project for this solution